README.md

```diff
@@ -24,11 +24,39 @@ It's a programming language designed for one purpose: to translate well to all t
 
 It's for writing libraries, and supporting other programming language communities with idiomatic, semantically equivalent libraries.
 
-Temper currently supports[\*](#not-endorsed-or-affiliated):<br>
-[![C#](docs/lang-icons/csharp.png)][C#] [<font color="#5382a1">Java</font>][Java] [![JavaScript](docs/lang-icons/js.png)][JS]/[![TypeScript](docs/lang-icons/ts.png)][TS] [![Lua](docs/lang-icons/lua.png)][Lua] [![Python](docs/lang-icons/py.png)][Python] [![Rust](docs/lang-icons/rust.png) Rust][Rust]
-
-Coming soon:<br>
-[![C++](docs/lang-icons/cpp.png)][C++], more.
+Temper currently supports[\*](#not-endorsed-or-affiliated):
+
+<table>
+  <tr valign="bottom">
+    <td align="center">
+      <a href="https://learn.microsoft.com/en-us/dotnet/csharp/"><img src="docs/lang-icons/csharp.png" alt="C#"><br>C#</a>
```

Comment:
I left the markdown-defined named links in the source for all the languages, even though I don't think they work for the html formatting. So I don't use the named links here and instead inline.